babel.config.js

```diff
@@ -24,6 +24,20 @@ module.exports = api => {
                 },
             ],
         ],
+        overrides: [
+            {
+                test: ['./src/**/*.ts', './src/**/*.tsx'],
+                presets: [
+                    [
```

Comment:
Do we need `@babel/preset-react` here?

Reply:
I don't think so, since it's already in the presets section. This is the overrides section for specific file types.

Reply:
Yeah, but from what i understand, if you add `@babel/preset-react`, you can rename current react files to `.ts` and `.tsx` right now. It's useful during your transition stage.

Reply:
Gotcha. I was really only looking at new files. If we find ourselves converting existing components, we should add it.